client/common/sass/components/_incident-database-card.sass

```diff
@@ -81,14 +77,28 @@
 
 		&--inline
 			display: block
-			padding-left: 19px
 			line-height: 0.5
 
+			+tablet-up
+				padding: 0.1rem 0
+				border: none
```

Comment:
I think let's keep the same style for mobile and tablet given the closed state design is same and we don't have a specific design for tablets.